src/util/metric.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use log::LogLevel;
+use cadence::prelude::{Counted, Timed, Gauged, Metered};
+use cadence::{StatsdClient, LoggingMetricSink};
+
+pub struct Metric {
```

Comment:
So I only need to add a new field `Arc<StatsdClient<LoggingMetricSink>>` in `Server`?

If so, I will delete the metric.rs, and use cadence directly.